pkg/gitreceive/build.go

```diff
@@ -178,14 +190,19 @@ func build(
 		)
 	} else {
 		buildPodName = slugBuilderPodName(appName, gitSha.Short())
+
+		cacheKey := ""
+		if !slugBuilderInfo.DisableCaching() {
+			cacheKey = slugBuilderInfo.CacheKey()
+		}
```

Comment:
I removed some logic in `slugbuilderPod`, so it only adds `cacheKey` to the env if it's non-empty, instead of checking the map.